inc/functions.php

```diff
@@ -901,6 +903,8 @@ function error($error_message = "", $title = "", $error_page = "")
 		echo json_encode(array("errors" => array($error_message)));
 		exit;
 	}
+	
+	http_response_code($status_code);
```

Comment:
Isn't `200` implied ? I'm unsure if the best practice is to always send a response code, so forgive my ignorance if so.

Reply:
It is indeed implied. I chose to set it explicitly here in case another part of the code might have modified the status code beforehand.

Reply:
I suppose plugins can update their usage of error() if so, then.